packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -48,23 +49,37 @@ class BlockListBlock extends Component {
 
 	getBlockForType() {
 		return (
-			<BlockEdit
-				name={ this.props.name }
-				isSelected={ this.props.isSelected }
-				attributes={ this.props.attributes }
-				setAttributes={ this.props.onChange }
-				onFocus={ this.onFocus }
-				onReplace={ this.props.onReplace }
-				insertBlocksAfter={ this.insertBlocksAfter }
-				mergeBlocks={ this.props.mergeBlocks }
-				onCaretVerticalPositionChange={
-					this.props.onCaretVerticalPositionChange
-				}
-				clientId={ this.props.clientId }
-				parentWidth={ this.props.parentWidth }
-				contentStyle={ this.props.contentStyle }
-				onDeleteBlock={ this.props.onDeleteBlock }
-			/>
+			<GlobalStylesContext.Consumer>
+				{ ( globalStyle ) => {
+					const mergedStyle = {
+						...globalStyle,
+						...this.props.wrapperProps.style,
+					};
+					return (
+						<GlobalStylesContext.Provider value={ mergedStyle }>
+							<BlockEdit
+								name={ this.props.name }
+								isSelected={ this.props.isSelected }
+								attributes={ this.props.attributes }
+								setAttributes={ this.props.onChange }
+								onFocus={ this.onFocus }
+								onReplace={ this.props.onReplace }
+								insertBlocksAfter={ this.insertBlocksAfter }
+								mergeBlocks={ this.props.mergeBlocks }
+								onCaretVerticalPositionChange={
+									this.props.onCaretVerticalPositionChange
+								}
+								wrapperProps={ this.props.wrapperProps }
```

Comment:
Do you think `wrapperProps` name is self descriptive enough? I want to stick to this name since it is coming from `getEditWrapperProps` but I doubt I'd understand this easily if I didn't know the context. It may be good to add an inline comments saying something like:

wrapperProps: block level styles
mergedStyle: inherited styles merged with block level styles

Reply:
Yeah, definitely it's worth to left a comment here :+1: